stage_1/Untitled.ipynb

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
What this file used for? There's no content. If not useful please delete it.

Reply:
Sorry, I'll delete it. It's automatically generated somehow ;(